dojo/settings/settings.dist.py

```diff
@@ -697,10 +710,10 @@ def generate_url(scheme, double_slashes, user, password, host, port, path, param
     SAML_LOGOUT_REQUEST_PREFERRED_BINDING = saml2.BINDING_HTTP_POST
     SAML_IGNORE_LOGOUT_ERRORS = True
     SAML_DJANGO_USER_MAIN_ATTRIBUTE = 'username'
-    # SAML_DJANGO_USER_MAIN_ATTRIBUTE_LOOKUP = '__iexact'
+    SAML_DJANGO_USER_MAIN_ATTRIBUTE_LOOKUP = '__iexact'
```

Comment:
Would be great to have this configurable and disable it by default, otherwise it may cause a security problem.

Reply:
Yes, sure my mistake.